### ISSUE TITLE
Docstring modification

### ISSUE THREAD

**Issue by caspervdw** [soft-matter/slicerator#20]
This started already in PR #18 (@lschr) How would we like to let `Slicerator` and `Pipeline` change the docstrings?

**Comment by danielballan** [comment]
@tacaswell has recently given some thought to this in matplotlib. I don't recall his thinking well enough to accurately summarize. Tom, if you have time, can you summarize and/or link to wherever docstring dicusssions are taking place in mpl?

**Comment by tacaswell** [comment]
https://github.com/matplotlib/matplotlib/issues/7179
